plugin-base/src/main/kotlin/io/spine/tools/gradle/project/ProjectExts.kt

```diff
@@ -122,3 +128,72 @@ public fun Project.configuration(name: String): Configuration = configurations.g
 /** Obtains a configuration by its name. */
 public fun Project.configuration(name: ConfigurationName): Configuration =
     configuration(name.value())
+
+/**
+ * Tells if this project can deal with Java code.
+ *
+ * @return `true` if `java` plugin is installed, `false` otherwise.
+ */
+public fun Project.hasJava(): Boolean = hasCompileTask(Java)
```

Comment:
Wouldn't it be simpler to use `Project.hasPlugin()`?

This method may cause tasks resolving.

Reply:
Yes. Good idea. I'll try and see if it works in the code of depending projects.